dist/assests/css/personal-training.css

```diff
@@ -0,0 +1,49 @@
+/* Start Personal Training Section Style*/
+.personal-training {
+  background-image: url(../img/image-personal-trainer.jpg);
+  background-size: cover;
+  background-repeat: no-repeat;
+  display: flex;
+  position: relative;
+  justify-content: center;
+  align-items: center;
+  height: 450px;
+  margin: 20px auto;
+}
+
+.personal-training .overlay {
+  width: 100%;
+  height: 100%;
+  left: 0;
+  top: 0;
+  background-color: #00000040;
+  position: absolute;
+}
+
+.personal-training .personal-training__container {
+  margin: 10px auto;
+  text-align: center;
+  color: #fff;
+  z-index: 1;
+  text-transform: uppercase;
+}
+
+.personal-training .personal-training__container .small-header {
+  font-size: 35px;
+  font-weight: 400;
+  line-height: 2;
+}
+
+.personal-training .personal-training__container .big-header {
+  font-size: 62px;
+  letter-spacing: 3px;
+  font-weight: 700;
+  line-height: 1;
+}
+
+.personal-training .personal-training__container button {
```

Comment:
two things
- Please use class selector instead of element selector whenever it is possible
- Why, in the first place, you are customising the button when there is a global btn which you can use?

Reply:
I remove all button styles and get the global styles